docs/deep-linking.md

```diff
@@ -40,24 +40,20 @@ We use a `fragment (code#)` for the activation code to avoid sending user data t
 Install: `npm i -g uri-scheme` if you want to use the simplified command
 
 ### a) Android
+
+#### Note
+For android trusted associations, which are required for https schemes, only work with signed apks out of the box.
+For local development you have to install the app and add these domains manually.
```

Comment:
I did not have to do this, it was working out of the box for me.

Reply:
hm were the trusted domains added there? Hm when i delete the apk on emulator and reinstall it they are not set by default. No idea why it worked but i would keep it in the docs even its working in some cases